tmpl/conversions.tmpl

```diff
@@ -85,26 +91,18 @@ func Strs(strs ...string) (cstrs **uint8, free func()) {
 		panic("Strs: expected at least 1 string")
 	}
 
-	// Allocate a contiguous array large enough to hold all the strings' contents.
-	n := 0
-	for i := range strs {
-		n += len(strs[i])
+	var pinned []string
+	var ptrs []*uint8
```

Comment:
Realized that this `Strs` implementation is problematic since this creates a Go pointer to a Go pointer, which is not permitted to pass to the C world.